open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java

```diff
@@ -0,0 +1,194 @@
+    {
+        GridPane thePane = new GridPane();
+        thePane.setMinWidth(width);
+        thePane.setMinHeight(height);
+        thePane.setStyle(
+                "-fx-background-color : #363636;-fx-border-color: #004080;-fx-border-width: 4;-fx-background-radius: 8 8 8 8;"
+                        + "    -fx-border-radius: 8 8 8 8;");
+
+        myDisplayArea = new GridPane();
+        ColumnConstraints col1Constraints = new ColumnConstraints();
+        col1Constraints.setPercentWidth(47);
+        ColumnConstraints col2Constraints = new ColumnConstraints();
+        col2Constraints.setPercentWidth(6);
+        ColumnConstraints col3Constraints = new ColumnConstraints();
+        col3Constraints.setPercentWidth(47);
+        myDisplayArea.getColumnConstraints().addAll(col1Constraints, col2Constraints, col3Constraints);
+        myDisplayArea.setVgap(5);
+        myDisplayArea.setPadding(new Insets(0, 10, 10, 10));
+        thePane.add(createHeader(header), 0, 0);
+        thePane.add(myDisplayArea, 0, 1);
+
+        return thePane;
+    }
+
+    public GridPane createBlankPane(String header)
+    {
+        GridPane thePane = new GridPane();
+        thePane.setStyle(
+                "-fx-background-color : #363636;-fx-border-color: #004080;-fx-border-width: 4;-fx-background-radius: 8 8 8 8;"
+                        + "    -fx-border-radius: 8 8 8 8;");
+
+        myDisplayArea = new GridPane();
+        ColumnConstraints col1Constraints = new ColumnConstraints();
+        col1Constraints.setPercentWidth(47);
+        ColumnConstraints col2Constraints = new ColumnConstraints();
+        col2Constraints.setPercentWidth(6);
+        ColumnConstraints col3Constraints = new ColumnConstraints();
+        col3Constraints.setPercentWidth(47);
+        myDisplayArea.getColumnConstraints().addAll(col1Constraints, col2Constraints, col3Constraints);
+        myDisplayArea.setVgap(5);
+        myDisplayArea.setPadding(new Insets(0, 10, 10, 10));
+        HBox topheader = createHeader(header);
+        GridPane.setHgrow(topheader, Priority.ALWAYS);
+        thePane.add(topheader, 0, 0);
+        thePane.add(myDisplayArea, 0, 1);
+
+        return thePane;
+    }
+
+    public GridPane getDisplayArea()
+    {
+        return myDisplayArea;
+    }
+
+    public void setDisplayArea(GridPane myDisplayArea)
+    {
+        this.myDisplayArea = myDisplayArea;
+    }
+
+    /**
+     * @param theLabel the text to set.
+     * @return mainHeader the HBox containing the sub window title.
+     */
+    public HBox createHeader(String theLabel)
+    {
+        HBox mainHeader = new HBox();
+        mainHeader.setStyle("-fx-background-color : #181a1c");
+        Label header = new Label(theLabel);
+        header.setFont(Font.font(16));
+        mainHeader.getChildren().add(header);
+        mainHeader.setAlignment(Pos.CENTER_LEFT);
+        HBox.setHgrow(header, Priority.ALWAYS);
+        return mainHeader;
+    }
+
+    /**
```

Comment:
Add explanation of method, no need to give the name of the return variable

Reply:
updated